cmd/group/main.go

```diff
@@ -127,7 +133,12 @@ func main() {
 		},
 	})
 
-	cmd.Flags().StringVar(&listen, "listen", listen, "listen address (unix or tcp) for the control endpoint")
+	log.Info("Adding cmd binds")
+	cmd.Flags().String("dir", discoveryDir, "Dir path for plugin discovery")
+	// Bind Pflags for cmd passed
+	viper.BindEnv("dir", "INFRAKIT_PLUGINS_DIR")
+	viper.BindPFlag("dir", cmd.Flags().Lookup("dir"))
+	cmd.Flags().String("name", name, "listen socket name for the control endpoint")
```

Comment:
Help text should be something like `"plugin name to advertise"`.  There's a handful of other identical lines that need the same change.